src/components/Comments/ReplyCommentItem.js

```diff
@@ -245,6 +329,7 @@ const ReplyCommentItem = ({
         </TouchableOpacity>
       </View>
 
+      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new BottomSheetMenu component has been added. This seems fine as long as the `refSheet`, `dataSheet`, and `height` props are properly defined and passed to the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->